operators/advanced_network/README.md

```diff
@@ -412,81 +341,56 @@ Too low means risk of dropped packets from NIC having nowhere to write (Rx) or h
 ```
 
 ##### Transmit Configuration (tx)
+
  (<mark>Current version of Rivermax manager doesn't support TX</mark>)
 
 - **`queues`**: List of queues on NIC
 	type: `list`
 	full path: `cfg\interfaces\tx\queues`
 	- **`name`**: Name of queue
   		- type: `string`
- 	- **`id`**: Integer ID used for flow connection or lookup in operator compute method
+	- **`id`**: Integer ID used for flow connection or lookup in operator compute method
   		- type: `integer`
-	- **`cpu_core`**: CPU core ID. Should be isolated when CPU polls the NIC for best performance.. <mark>Not in use for Doca GPUNet IORivermax manager</mark>
+	- **`cpu_core`**: CPU core ID. Should be isolated when CPU polls the NIC for best performance.. <mark>Not in use for Doca GPUNetIO</mark>
```

Comment:
Not related to this review but I thought this would still apply. Even with GPUNetIO you need the CPU waiting to see if packets arrived so it could do something.

Reply:
idk. let's discuss offline with elena. she mentioned in the past some things to not about the cpu core (like it would ignore if you give it different values?)